Remove call to METIS_PartMesh*, and return an error if the user tries it.

Index: elmerfem/elmergrid/src/femelmer.c
===================================================================
--- elmerfem.orig/elmergrid/src/femelmer.c
+++ elmerfem/elmergrid/src/femelmer.c
@@ -2458,6 +2458,9 @@
   int *neededby,*metistopo;
   int *indxper,*inpart,*epart,*npart;
 
+  printf ("Element partitioning not supported by the Debian package\n");
+  return 1;
+
   if(info) printf("Making a Metis partitioning for %d elements in %d-dimensions.\n",
 		  data->noelements,data->dim);
 
@@ -2562,6 +2565,7 @@
 
   if(info) printf("Using %d nodes of %d possible nodes in the Metis graph\n",nn,noknots);
 
+  /*
   if(dual) {
     if(info) printf("Starting graph partitioning METIS_PartMeshDual.\n");  
     METIS_PartMeshDual(&ne,&nn,metistopo,&etype,
@@ -2572,6 +2576,7 @@
     METIS_PartMeshNodal(&ne,&nn,metistopo,&etype,
 			&numflag,&nparts,&edgecut,epart,npart);
   }
+  */
 
   /* Set the partition given by Metis for each element. */
   for(i=1;i<=noelements;i++) {
